test/known_issues/known_issues.status

```diff
@@ -18,3 +18,7 @@ test-vm-timeout-escape-promise: PASS,FLAKY
 [$system==freebsd]
 
 [$system==aix]
+
+[$arch==arm]
+# https://github.com/nodejs/node/issues/24120
+known_issues/test-vm-timeout-escape-nexttick: PASS,FLAKY
```

Comment:
Is the entry respected when it includes the directory name like this? (I honestly don't know. We don't use it elsewhere, so regardless, for consistency, perhaps we shouldn't use it here?)

```suggestion
test-vm-timeout-escape-nexttick: PASS,FLAKY
```

Reply:
@Trott - thanks, fixed.